turtleGraphics.js

```diff
@@ -0,0 +1,107 @@
+class Turtle {
+    constructor (x, y) {
+        this.x = x;
+        this.y = y;
+        this.direction = "east";
+        this.points = [[x, y]];
+    }
+    
+    forward (n) {
```

Comment:
Subjective criticism alert:

I would try to keep every function to have one single function. So the forward, would be the one that triggered a `move_north` or `move_south` method. The reason are as follows:

The forward method is in charge of moving the turtle everywhere so it makes the function verbose. Instead of having all these lines 29 lines you could have a method call within each if block like so:

```
...
if (this.direction === "north") {
  this.move_north(n);
} else if (this.direction === "east") {
  this.move_east(n);
} else if (this.direction === "south") {
  ...
```

turning your forward into a router, and each method into the logic that moves the turtle into one specific direction.

Advantages:

* each method is independent and won't affect each other
* when testing, you can even test the forward as a router and each directional function independently

AGAIN, THIS IS SUBJECTIVE. The way you did it is great.